src/app/components/history-table/history-table.component.html

```diff
@@ -123,20 +123,27 @@
                     *ngIf="getUserToken(event) as userToken"
                     class="table__amount"
                 >
-                    <ng-container *ngIf="isReceivedEvent(event); else minus">
-                        +
-                    </ng-container>
-                    <ng-template #minus>
-                        -
-                    </ng-template>
-                    <span
-                        *ngIf="
-                            event.amount | decimal: userToken.decimals as amount
-                        "
-                        [matTooltip]="amount"
+                    <ng-container
+                        *ngIf="event.event !== 'EventPaymentSentFailed'"
                     >
-                        {{ amount | displayDecimals: 3 }}
-                    </span>
+                        <ng-container
```

Comment:
Out of curiosity could you do this also like this? `{{ isReceivedEvent(event) ? '+' : '-' }}`

Reply:
Yes you can. And that is way more readable. I will go for it.